app/src/main/res/values-de-land/arrays.xml

```diff
@@ -24,7 +24,6 @@
         <item>Ö</item>
         <item>Ä</item>
         <item>Y</item>
-        <item>\'</item>
```

Comment:
Does this need to be removed from the other German files?

Reply:
It looks like this was a duplicate character in the values-de-land file only.